README.md

```diff
@@ -119,6 +131,12 @@ To run in a production environment, it is easiest to use `docker-compose`:
 
     ```bash
     docker-compose up -d
+
+    # use minio yml
+    docker-compose -f ./docker-compose.minio.yml up -d
+    
+    # use local disk yml
+    docker-compose -f ./docker-compose-local.yml up -d
```

Comment:
Can `docker-compose-local.yml` be renamed to `docker-compose.local.yml` so that it matches the naming scheme of the `minio` yml file?

Reply:
solved.